content/200-concepts/100-components/04-introspection.mdx

```diff
@@ -175,6 +146,22 @@ model User {
 }
 ```
 
+Trying to generate your Prisma Client with `prisma generate` you would get the following error:
+
+```
+$ npx prisma generate
```

Comment:
Here would be a good place to use the `CodeWithResult` component. I tried to do a suggestion but the component includes backticks and that confused github 😅. Here is a link to the component and how to use it: https://www.prisma.io/docs/about/style-guide/mdx-examples#code-with-output

Reply:
You can just use 4 backticks for the suggestion code if you need to wrap something that has three backticks itself.

Reply:
https://github.com/prisma/docs/pull/1667/files#r687864653